### DISCUSSION TITLE
[질문] remote config를 위한 dependecny 추가

### DISCUSSION THREAD

**Discussion by boogi-woogi** [TEAM-DAME/Dame-Android#71] [2. Q&A] [answered]
![image](https://user-images.githubusercontent.com/88770426/202246090-71ebf990-e0e8-49d5-a82e-e63f98b358ab.png)
remote config를 활용하기 위해서는 "com.google.firebase:firebase-config-ktx" 를 dependencies에 추가해야 하는 것을 알게되었습니다. 

![image](https://user-images.githubusercontent.com/88770426/202246307-14ccc196-a8aa-4a87-90da-66aea89f4fa9.png)
위 사진에 나오는 notification의 build.gradle파일에 해당 dependency를 추가해도 되는지 궁금합니다!

**Comment by onseok** [comment, marked as answer]
해당 사항 반영하여 develop 브랜치에 최신화 해두었습니다!
